scripts/test-policies.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Does this file run on the CI? Would that be good?

Reply:
Yes, meant to add this to the test, thanks for pointing out